src/appShell/App/PortalHeader.tsx

```diff
@@ -90,6 +90,14 @@ export default class PortalHeader extends React.Component<
                 hide: () =>
                     AppConfig.serverConfig.skin_show_about_tab === false,
             },
+
+            {
+                id: 'installation-map',
+                text: 'cBioPortal Installations',
+                address: '/installations',
+                internal: false,
+                hide: () => AppConfig.serverConfig.app_name !== 'public-portal',
```

Comment:
Maybe we should add a configuration in the properties file (turn off by default)? @alisman

Reply:
@jtquach1 yeah we're trying not to use the name property like this.  reach out to me and i can tell you how to add configuration

Reply:
@alisman would this be done in `src/config/serverConfigDefaults.ts`?